mesonbuild/mesonlib.py

```diff
@@ -881,6 +881,40 @@ def windows_proof_rmtree(f):
     # Try one last time and throw if it fails.
     shutil.rmtree(f)
 
+def unholder_array(entries):
+    result = []
+    entries = flatten(entries)
```

Comment:
__[Flake8]__

undefined name 'flatten'


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
This flake8 issue seems to be important.

Reply:
This seems to be dead code. Removed.